mobile/lib/src/main.dart

```diff
@@ -1,115 +1,84 @@
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug painting" (press "p" in the console, choose the
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
-            ),
-          ],
-        ),
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
-    );
+        debugShowCheckedModeBanner: false,
+        title: 'Aeris',
+        localizationsDelegates: const [
+          GlobalMaterialLocalizations.delegate,
+          GlobalWidgetsLocalizations.delegate,
+          GlobalCupertinoLocalizations.delegate,
+          FormBuilderLocalizations.delegate
+        ],
+        supportedLocales: const [Locale('en', ''), Locale('fr', '')],
+        theme: ThemeData(colorScheme: aerisScheme),
+        initialRoute: '/home',
+        onGenerateRoute: (settings) {
+          Map pageRoutes = {
+            '/': () => const StartupPage(),
+            '/login': () => const LoginPage(),
+            '/home': () => const HomePage(),
+          };
+          Map cardRoutes = {
+            '/pipeline': () => const PipelineDetailPage(),
+            '/services': () => const ServicePage(),
+            '/pipeline/action/mod': () => const SetupActionPage(),
+            '/pipeline/action/new': () => const SetupActionPage(),
+            '/pipeline/new': () => const CreatePipelinePage()
+          };
+          Map routes = {}
+            ..addAll(cardRoutes)
+            ..addAll(pageRoutes);
+          return PageRouteBuilder(
+              opaque: false,
+              settings: settings,
+              pageBuilder: (_, __, ___) => routes[settings.name].call(),
+              transitionDuration: const Duration(milliseconds: 500),
```

Comment:
I don't know what transitions look like are, but 500ms seems a lot for animations

Reply:
## Test method

I'm testing values between 225ms and 800ms to check which one is the best looking. I'll let you know.

Reply:
Animation duration value has been modified to 350ms.